xt/66-cucumber/README.md

```diff
@@ -136,11 +135,11 @@ and the single-page app) are identified through the 'id' attribute of the
 
 The single page app's DOM tree:
 
-  * Weasel::Session's `page` attribute (PageObject::Root)
+* Weasel::Session's `page` attribute (PageObject::Root)
     * `body` attribute (PageObject::App)
-      * `menu` attribute (PageObject::App::Menu)
-      * `maindiv` attribute (PageObject::App::Main)
-        * `content` attribute (PageObject::App::*)
+        * `menu` attribute (PageObject::App::Menu)
+        * `maindiv` attribute (PageObject::App::Main)
+            * `content` attribute (PageObject::App::*)
```

Comment:
Sub lists should only be indented by 2 spaces per level.
[Refer to Github Markdown Syntax Guide](https://guides.github.com/features/mastering-markdown/)

Reply:
Fixed